packages/@ember/-internals/glimmer/lib/utils/managers.ts

```diff
@@ -17,6 +18,8 @@ const COMPONENT_MANAGERS = new WeakMap<
   ManagerFactory<ComponentManagerDelegate<unknown> | InternalComponentManager>
 >();
 
+const FROM_CAPABILITIES = DEBUG ? new WeakSet() : undefined;
```

Comment:
Doesn't look like the `WeakSet` polyfill is being used here, may be why browserstack is failing

Reply:
Nice, thanks for catching that!